assets/stylesheets/base/_mixins.scss

```diff
@@ -0,0 +1,9 @@
+@mixin border-bottom($width: 1px, $style: solid, $color: $underline-color) {
```

Comment:
Not a huge fan of this... what value does this mixin provide? Feels like something else I need to remember.